src/cpp/src/speculative_decoding/continuous_batching_for_speculative_decoding_impl.cpp

```diff
@@ -218,8 +214,6 @@ UpdateRequestResult
 ContinuousBatchingPipeline::ContinuousBatchingForSpeculativeDecodingImpl::update_request(uint64_t request_id,
                                                                                          const GeneratedSequences& candidates,
                                                                                          bool is_update_logit_processor) {
-    _pull_awaiting_requests();
```

Comment:
we need to stay this line here because in `draft_model` requests will be running, and in `main_model` they be awaiting...